### PR TITLE
Show report for openzeppelin run (based on stdout)

### PR DESCRIPTION
**Pull request by gigimon**
1. Parse stdout from parallel and show test statistics based on it
2. Fix allure report (setup feature for openzeppelin-contract tests)

Report in shell: https://github.com/neonlabsorg/neon-compatibility/runs/4618960601?check_suite_focus=true
Small allure report (before update code to use faucet, small count of tests): https://docs.neon-labs.org/neon-compatibility/feature/test-report/318/#

Full allure report will in build: https://github.com/neonlabsorg/neon-compatibility/actions/runs/1615953619
https://docs.neon-labs.org/neon-compatibility/feature/test-report/319/#

### CODE REVIEW

File: .github/workflows/consecutive.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ on:
         type: choice
         default: self-hosted
         required: true
-        description: "Choose out of (self-hosted, ubuntu-20.04)"
+        description: "Where run openzeppelin tests (self-hosted, ubuntu-20.04)"
```
Comment (apetrovskiy) [inline .github/workflows/consecutive.yml]: to run?

File: .github/workflows/consecutive.yml [hunk 2]
```diff
@@ -146,15 +146,10 @@ jobs:
           find test -name "*.test.js" | sort | parallel -k --jobs ${{ env.JOBS_NUMBER }} --results pout 'npx hardhat test {}'
       - name: "Verify count of logs"
         if: always()
-        run: |
-          cd openzeppelin-contracts
-          echo "Tests by find:" && find test -name "*.test.js" | sort | wc -l
-          echo "Test results:" && ls pout/1 | wc -l
-      - uses: ./.github/actions/allure-postprocessing
```
Comment (apetrovskiy) [inline .github/workflows/consecutive.yml:153]: ?

Comment (gigimon) [reply .github/workflows/consecutive.yml:153]: because fix below (work without additional plugin, based on json)

Comment (apetrovskiy) [reply .github/workflows/consecutive.yml:153]: image

**Review by apetrovskiy — DISMISSED** [review]

File: .github/workflows/consecutive.yml [hunk 3]
```diff
@@ -287,8 +282,6 @@ jobs:
     needs:
       - allure
       - prepare-env
-    env:
-      NETWORK: ${{ needs.prepare-env.outputs.network }}
     steps:
       - name: Send a notification to Teams channel
         uses: neonlabsorg/teams-notification-action@0.1.1
```
Comment (apetrovskiy) [inline .github/workflows/consecutive.yml]: `uses: neonlabsorg/teams-notification-action@1.2`
however, prefer avoiding of adding everything in a single PR

File: .github/workflows/consecutive.yml [hunk 4]
```diff
@@ -23,7 +23,7 @@ on:
         type: choice
         default: self-hosted
         required: true
-        description: "Choose out of (self-hosted, ubuntu-20.04)"
+        description: "Where to run openzeppelin tests (self-hosted, ubuntu-20.04)"
```
Comment (apetrovskiy) [inline .github/workflows/consecutive.yml:26]: Technically, it's not about where to run, it's about what type of runner to use
Network combo box is more close to 'where'
btw, isn't the proposed label too long?

**Review by apetrovskiy — APPROVED** [review]

**Comment by gigimon** [comment]
https://docs.neon-labs.org/neon-compatibility/feature/test-report/333/ last result, 1800+

**Pull request merged**